### PR TITLE
[Generic Scraper] Added the article API

### PR DESCRIPTION
**Pull request by daminisatya**
#598 
Leveraging NLP from Boilerpipe for article extraction

### CODE REVIEW

**Comment by daminisatya** [comment]
@sudheesh001 Looks like the codacy is not able to trace the code

File: .classpath [hunk 1]
```diff
@@ -68,7 +68,7 @@
 	<classpathentry kind="lib" path="lib/jsr166e-1.1.0.jar"/>
 	<classpathentry kind="lib" path="lib/netty-3.10.5.Final.jar"/>
 	<classpathentry kind="lib" path="lib/t-digest-3.0.jar"/>
-	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/J2SE-1.5"/>
 	<classpathentry kind="lib" path="conf/logs"/>
```
Comment (shivenmian) [inline .classpath]: What's this for?

**Comment by sudheesh001** [comment]
Yes @daminisatya Codacy can't check such huge requests, anyway codacy isn't an issue here. This patch looks good, but please make the changes suggested to you in the discussion

**Comment by daminisatya** [comment]
http://localhost:9000/api/genericscraper.json?url=http://stackoverflow.com/questions/15655012/how-final-keyword-works&type=article

Sample query.

**Comment by daminisatya** [comment]
selection_309

**Pull request merged**